### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by VedanthB**
- added home page components
- added eslint + prettier 
- added routes

### CODE REVIEW

**Review by Richa-rani02 — APPROVED** [review]
suggestion: U can raise separate PR for initial files and folder structure or u can mention in PR which file to check

File: src/App.css [hunk 1]
```diff
@@ -1,51 +0,0 @@
...
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
-}
-
-.brand-title {
-  font-size: 3rem;
-  color: #d9d9d9;
```
Comment (Ananyamadhu08) [inline src/App.css:32]: can use variables for colors

File: src/frontend/styles/landing-page.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+.featuredCategories__container {
+  margin-top: 30px;
+  padding: 30px 0px 26px;
+  display: grid;
+  grid-gap: 25px;
+  gap: 25px;
+  grid-template-columns: repeat(5, minmax(0, 1fr));
+}
+
+/* @media (max-width: 768px) {
```
Comment (Ananyamadhu08) [inline src/frontend/styles/landing-page.css:31]: can delete unnecessary commented code

**Review by Ananyamadhu08 — APPROVED** [review]
Good job Vedanth! you can delete the commented code, other than that your code is neat and structured! You can go ahead and merge 🚀

**Pull request merged**